sema/commons/ogm/graph_wrapper.py

```
@@ -0,0 +1,125 @@
+
+from .term_builder import TermBuilder
+
+
+class GraphWrapper:
+    def __init__(self, graph=None, base=None):
+        self.graph: Graph = graph or Graph(base=base, bind_namespaces="none")
+        self._listify = lambda x: x if isinstance(x, list) else [x]
+
+    def set_jsonld_context(self, context) -> None:
+        self.graph._jsonld_context = Context(context)
+        self.graph._jsonld_context_repr = context  # Context.to_dict() doesn't preserve original representation # noqa: E501
```

Ruff (0.14.13)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)